generatebundlefile/ecr_helper.go

```diff
@@ -186,7 +186,16 @@ func getLatestHelmTagandSha(details []ImageDetailsBothECR) (string, string, erro
 	if reflect.DeepEqual(latest, ImageDetailsBothECR{}) {
 		return "", "", fmt.Errorf("error no images found")
 	}
-	return latest.ImageTags[0], *latest.ImageDigest, nil
+
+	var imageTag string
+	imageTagRegex := regexp.MustCompile(`^\d+\.\d+\.\d+-[0-9a-f]{40}$`)
```

Comment:
Does this handle eks-d tag that follows a version. Not sure what this regex is trying to capture but tried running it online against a sample tag we might have for metrics server: `9.37.0-1.27-d18f9e2def8e8e24b3ce0e0c543cc4d1d7d6512e` and it failed.

Reply:
good catch, fixed the regex